### ISSUE TITLE
ability to import any paste id

### ISSUE THREAD

**Issue by bhanderson** [tucked/pbnh#5]
if you are migrating from haste it has an id and content

**Comment by jetpacktuxedo** [comment]
This should work on the db end.

Do we have any way of accessing the haste/paste dbs directly? I can throw something together to handle the import.

**Comment by bhanderson** [comment]
yeah I have a db. Unfortunately some indexes are numbers and some are 10 char strings

**Comment by jetpacktuxedo** [comment]
We can set the 10 char string as the "hash" in the new database. Then lookups will still work, but we won't be able to detect duplicates.

**Comment by tucked** [comment]
I'm not really familiar with haste, but the recommended way to import pastes now is via command line:
```sh
curl https://other.paste.bin/abc123 | curl --form content=@- https://pbnh.example.com
```
